src/drivers/fs-lite.ts

```diff
@@ -73,8 +73,8 @@ export default defineDriver((opts: FSStorageOptions = {}) => {
       }
       return unlink(r(key));
     },
-    getKeys() {
-      return readdirRecursive(r("."), opts.ignore);
+    getKeys(_base, { maxDepth }) {
```

Comment:
Do you mind refactoring it to pass whole `tOpts` to underlying utils? Reason: Avoid ref creation by destruct and also pass any future keys.